Modules/mathmodule.c

```diff
@@ -3514,6 +3514,29 @@ static uint64_t inverted_factorial_odd_part[] = {
     0x547fb1b8ab9d0ba3u, 0x8f15a826498852e3u, 0x32e1a03f38880283u, 0x3de4cce63283f0c1u,
 };
 
+/* exponent of the largest power of 2 dividing factorial(n), for n in range(68)
+
+Python code to generate the values:
+
+import math
+
+for n in range(68):
+    fac = math.factorial(n)
+    fac_trailing_zeros = (fac & -fac).bit_length() - 1
+    print(f"{fac_trailing_zeros}u")
+*/
+
+static uint8_t factorial_trailing_zeros[] = {
+    0u, 0u, 1u, 1u, 3u, 3u, 4u, 4u,
```

Comment:
Is the `u` suffix necessary?

Reply:
Probably not: it was a pre-emptive move to silence possible compiler complaints about signed-to-unsigned conversion. I've removed the "u"s - we can re-instate if compilers think they need it.